src/gwt/src/org/rstudio/core/client/theme/res/themeStyles.css

```diff
@@ -401,6 +405,10 @@ pre {
    width: 7px;
 }
 
+.rstudio-themes-flat.rstudio-themes-dark-menus .gwt-MenuBar .subMenuIcon img {
+   filter: invert(100%);
```

Comment:
CSS is scoped to the image and the `subMenuIcon` class is specific to the array. Yes, supported in QTWebKit: https://caniuse.com/#search=css%20filter

<img width="339" alt="screen shot 2017-05-26 at 3 39 21 pm" src="https://cloud.githubusercontent.com/assets/3478847/26514839/86bfde7c-4229-11e7-8ecc-3f84e2182a9c.png">